Tests/UserSessionTests.swift

```diff
@@ -81,6 +85,40 @@ class UserSessionTests: XCTestCase {
         })
     }
     
+    func test_RefreshUserSession_RefreshMultipleSessions() {
+        userSession?.refreshHandler = refreshHandler(_:)
+        let allSessions: [Refreshable] = [userSession!, userSession!]
+        
+        let expectation = XCTestExpectation(description: "All sessions refresh")
+        allSessions.refresh { (_) in
+            //Since we're really just refreshing the same session twice, this should be 2
+            XCTAssertEqual(self.refreshCallCount, 2)
+            expectation.fulfill()
+        }
+        
+        wait(for: [expectation], timeout: 10.0)
```

Comment:
Since there aren't any actual network calls going on, is there any reason we can't go ahead and make this timeout 1 second instead of 10?

Reply:
No reason -- Changing to 1.